src/OrchardCore.Modules/OrchardCore.Title/Drivers/TitlePartDisplay.cs

```diff
@@ -47,12 +39,5 @@ public override async Task<IDisplayResult> UpdateAsync(TitlePart model, IUpdateM
 
             return Edit(model);
         }
```

Comment:
Yes, this looks like a good idea.

But here where you change the signature of the `Edit` method you must return the same method signature in `UpdateAsync` or the updating and validation breaks.

We could probably update the templates to use the same technique.

Reply:
done.